lib/plugins/aws/info/getStackInfo.js

```diff
@@ -57,6 +44,24 @@ module.exports = {
       }
 
       return BbPromise.resolve();
+    })
+    .then(() => this.provider.getAccountId())
+    .then((accountId) => {
+      const stage = this.provider.getStage();
+      const region = this.provider.getRegion();
```

Comment:
If I understand this correctly, this is based on the region option, along with the `us-east-1` default. So the generated ARNs don't reflect what is actually deployed. So if I only deploy to `us-west-2` and then run `sls info`, that would generate ARNs pointing to none existing functions in `us-east-1`. Is that correct?